resources/views/settings/ldap.blade.php

```diff
@@ -63,64 +63,44 @@
                                 {{ Form::label('ldap_integration', trans('admin/settings/general.ldap_integration')) }}
                             </div>
                             <div class="col-md-9">
-                                @if (config('app.lock_passwords')===true)
-                                    {{ Form::checkbox('ldap_enabled', '1', Input::old('ldap_enabled', $setting->ldap_enabled),['class' => 'minimal disabled', 'disabled' => 'disabled']) }}
-                                @else
-                                    {{ Form::checkbox('ldap_enabled', '1', Input::old('ldap_enabled', $setting->ldap_enabled),array('class' => 'minimal')) }}
-                                @endcan
-
+                                {{ Form::checkbox('ldap_enabled', '1', Input::old('ldap_enabled', $setting->ldap_enabled), ['class' => 'minimal '. $setting->demoMode, $setting->demoMode]) }}
                                 {{ trans('admin/settings/general.ldap_enabled') }}
                                 {!! $errors->first('ldap_enabled', '<span class="alert-msg">:message</span>') !!}
                             </div>
                         </div>
 
-                        <!-- AD Flag -->
```

Comment:
It definitely makes sense to flip these two controls, but again - would rather see that in a separate PR. Though, maybe I'm wrong here - since this is a major release, maybe UI changes like this ought to happen now rather than having to wait until much later.